modules/ocf_kubernetes/manifests/loadbalancer.pp

```diff
@@ -0,0 +1,38 @@
+class ocf_kubernetes::loadbalancer {
+  include ::haproxy
+
+  $kubernetes_workers_ipv4 = lookup('kubernetes::workers_ipv4')
+  $kubernetes_worker_nodes = lookup('kubernetes::worker_nodes')
+
+  haproxy::frontend { 'kubernetes-frontend':
+    mode    => 'http',
+    bind    => {
+      "${::ipaddress}:80"   => [],
+      # TODO: Listen on 443 when we add TLS
+      #"$::ipaddress:443"  =>  ['ssl', 'crt', '/path/to/cert.pem']
+    },
+    options => {
+      'default_backend'  => 'kubernetes-backend',
+      # TODO: enable once we add TLS
+      #'redirect' =>  'scheme https code 301 if !{ ssl_fc }',
+    };
+  } ->
+  haproxy::backend { 'kubernetes-backend':
+    options => {
+      option         => [
+        'forwardfor'
+      ],
+      'mode'         => 'http',
+      'balance'      => 'roundrobin',
```

Comment:
will this work with sessions?

Reply:
I think so. This is just roundrobining to ingress nginx, which will handle [session affinity](https://github.com/kubernetes/ingress-nginx/tree/master/docs/examples/affinity/cookie)